cmd/influxd/launcher/launcher.go

```diff
@@ -178,7 +178,6 @@ var vaultConfig vault.Config
 
 func setLauncherCMDOpts(l *Launcher, cmd *cobra.Command) {
 	cli.BindOptions(cmd, launcherOpts(l))
-	cmd.AddCommand(inspect.NewCommand())
```

Comment:
Before this, the code was registering both `influxd inspect` and `influxd run inspect` commands. That felt like a mistake, but this technically a breaking change.  I felt decently confident making the change because the public docs site only list `influxd inspect`, but I can revert if needed.

Reply:
This is good with me; `run` shouldn't have any subcommands.